pkg/time/time_skew_amd64.go

```diff
@@ -294,12 +294,12 @@ func (it *CompositeInjector) Recover(pid int) error {
 	return nil
 }
 
-func NewCompositeInjector(deltaSec int64, deltaNsec int64, clockIdsMask uint64) (*CompositeInjector, error) {
-	SkewClockGetTime, err := NewTimeSkew(deltaSec, deltaNsec, clockIdsMask)
+func NewTimeSkew(deltaSec int64, deltaNsec int64, clockIdsMask uint64) (FakeClockInjector, error) {
+	SkewClockGetTime, err := NewSkewClockGetTime(deltaSec, deltaNsec, clockIdsMask)
```

Comment:
```suggestion
	skewClockGetTime, err := NewSkewClockGetTime(deltaSec, deltaNsec, clockIdsMask)
```
please use `camelCase` instead of `CamelCase` for local variable.